Server/controllers/articles.controller.js

```diff
@@ -19,4 +19,20 @@ exports.getAllArticles = (req, res, next) => {
         res.status(200).send( {articles} )
     })
     .catch( (err) => { next(err) })
-}
\ No newline at end of file
+}
+
+exports.patchArticleByID = (req, res, next) => {
+
+    const article_id = req.params.article_id
+    const inc_votes = req.body.inc_votes
+   
+    if (inc_votes === undefined) {
+        return next({ status: 400, msg: "no votes submitted" })
+    }
+
+    updateArticleByID(article_id, inc_votes)
+    .then( article => {
+        res.status(202).send({ article })
+    })
+    .catch( err => next(err) )
```

Comment:
If you like there is a shorthand syntax for the catch 
`.catch(next)` which will invoke next with the error